tests/server/metrics.js

```diff
@@ -37,6 +37,7 @@ suite.tests['#post /metrics - returns 200 with valid data'] = {
       testValidMetricsEvent('type', 'error.unknown context.auth.108'),
   'valid error-type (signin-permissions.checkbox.change.profile:display_name.unchecked)':
     testValidMetricsEvent('type', 'signin-permissions.checkbox.change.profile:display_name.unchecked'),
+  // 'valid experiment choice (sendSms)': testValidMetricsField('experiments', { choice: 'sendSms', group: 'treatment'}),
```

Comment:
You were on the right lines here, just the value needs to be an array. So pop some square brackets in and reinstate this line.

Oh, and great catch! Because we were missing a test for the valid case like this oen, we'd never realised that our tests for the invalid case were broken.